main.js

```diff
@@ -0,0 +1,74 @@
+// Tracks Board State //
+let gameBoardObj = {
+  1: [5, 4, 3, 2, 1],
+  2: [],
+  3: []
+}
+
+
+
+
+// Handles Opening Display of Game Board //
+let renderGame = function() {Object.values(gameBoardObj).map((game) => {
```

Comment:
for readability I would store Object.values(gameBoardObj) in a const and then do the forEach on that const.